roles/edpm_nova/templates/02-nova-host-specific.conf.j2

```diff
@@ -4,6 +4,12 @@ host = {{ canonical_hostname }}
 
 [libvirt]
 live_migration_with_native_tls = {{ edpm_nova_live_migration_native_tls|bool }}
+
 {% if edpm_nova_live_migration_tls|bool %}
-live_migration_scheme = tls
+live_migration_uri = qemu+tls://%s/system
+{% endif %}
+
+{% if edpm_nova_live_migration_native_tls|bool %}
+# FIXME(owalsh): workaround https://issues.redhat.com/browse/LIBVIRT-1113
+live_migration_permit_post_copy = False
```

Comment:
this is a regression form 17.1 so we should try and address this before GA

Reply:
that is up to the libvirt team, as I can see from the linked issue. Also VIR_MIGRATE_PARALLEL feature is affected